guides/doc-Planning_for_Project/topics/Required_Technical_Users.adoc

```diff
@@ -25,14 +25,20 @@ Because of potential conflicts with local users that {Project} creates, you cann
 |User name |UID |Group name |GID |Flexible UID and GID |Home |Shell
 |foreman |N/A |foreman |N/A |yes |/usr/share/foreman |/sbin/nologin
 |foreman-proxy |N/A |foreman-proxy |N/A |yes |/usr/share/foreman-proxy |/sbin/nologin
+|apache |48 |apache |48 |no |/usr/share/httpd |/sbin/nologin
+|postgres |26 |postgres |26 |no |/var/lib/pgsql |/bin/bash
+ifdef::katello,orcharhino,satellite[]
 |pulp |N/A |pulp |N/A |no |N/A |/sbin/nologin
+endif::[]
 |puppet |N/A |puppet |N/A |yes |/opt/puppetlabs/server/data/puppetserver |/sbin/nologin
```

Comment:
Since puppetserver 6.20.0 / 7.8.0 this user is (once again) created with uid 52, like it did in the Puppet < 4 days. It even has an official reservation (though with a different home directory):
```console
$ grep puppet /usr/share/doc/setup/uidgid 
puppet	52	52	/var/lib/puppet		/sbin/nologin	puppet
```
https://github.com/puppetlabs/puppetserver/commit/7474225cd475d43ac4beb00bacc494c1619e0f7f / https://tickets.puppetlabs.com/browse/SERVER-1381